docs/spec/spec.md

```diff
@@ -58,33 +70,49 @@ broker._
 
 ## kind: Broker
 
-### group: eventing.knative.dev/v1beta1
+### group: eventing.knative.dev/v1
 
-_A Broker represents an event mesh. It logically receives events on its input
-domain and forwards them to subscribers defined by one or more matching
-Trigger._
+_A `Broker` collects a pool of events that are consumable using triggers.
+Brokers provide a well-known endpoint for event delivery that senders can use
```

Comment:
I wonder if we should change 'well-known' to a discoverable endpoint (status.address).

Reply:
Updated.